package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "blockreassurance",
+  "version": "1.0.0",
+  "description": "## About",
+  "private": true,
+  "scripts": {
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/PrestaShop/blockreassurance.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "AFL",
+  "bugs": {
+    "url": "https://github.com/PrestaShop/PrestaShop/issues"
+  },
+  "homepage": "https://github.com/PrestaShop/blockreassurance#readme",
+  "devDependencies": {
+  },
+  "dependencies": {
+  }
```

Comment:
Missing important information:

- author
- platform requirements (php version)
- type: "prestashop-module"

You can use this one as an example: https://github.com/PrestaShop/autoupgrade/blob/master/composer.json

Reply:
@eternoendless You compare a composer.json and a package.json ?

Reply:
Author :ok:

Reply:
type & platform-requirements (or platform or requirement(s)) doesn't exist on package.json : https://docs.npmjs.com/files/package.json